openduck-py/openduck_py/voices/piper.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import torch
+from torchaudio.functional import resample
+
+from piper.voice import PiperVoice
+
+from .settings import DEVICE
+
+model = PiperVoice.load(
+    "models/cartoon-boy-upbeat-piper.onnx",
+    config_path="models/cartoon-boy-upbeat-piper.onnx.json",
+    use_cuda=(DEVICE == "cuda"),
+)
+
+
+def inference(
+    text: str, output_sample_rate: int, language: str = "english"
+) -> np.ndarray:
+    audio = model.synthesize_stream_raw(
+        text,
+        speaker_id=0,
+    )
+    audio = b"".join(audio)
+    audio = torch.frombuffer(audio, dtype=torch.int16).float() / 32767  # TODO silly
```

Comment:
should it be / 32768 ? (2^15)
not 32767?

not a big difference though

also whats # TODO silly?

Reply:
it's silly because i'm undoing [the conversion piper does](https://github.com/rhasspy/piper/blob/master/src/python_run/piper/util.py) (which uses 32767 btw)